### ISSUE TITLE
Three broken boards

### ISSUE THREAD

**Issue by JsGraaf** [sparkfun/QuickLogic_QTPlus#2]
Hello, 

I have three QuickLogic Thing Plusses and none of them work. Two of the three boards give a blue flashing light which turns off after 5 seconds but they can't enter programming mode (no green breathing light after pressing the usr button within 5 seconds). The other board shows a red led when powering up, which promply turns off. I have tried reflashing the bootloader with openocd and an STLinkv2, as stated on https://forum.quicklogic.com/viewtopic.php?t=29, however, the STLink is unable to restart the board after programming, failing the entire operation. I have not been able to find any resources from Sparkfun on recovering these boards and there is a "dead" thread on the forum that never got a reply.

I'm hoping that someone here can help.

**Comment by Funda** [comment]
JsGraaf, thank you for reaching out here. I am investigating the issue today and will hopefully have a response to the forum thread by the end of the day. I see you replied to the thread. Let's continue the conversation there: https://forum.sparkfun.com/viewtopic.php?f=176&t=56658&p=238814&hilit=QuickLogic#p238814

Thank you!

**Comment by JsGraaf** [comment]
Using a JLink mini EDU, I was able to reflash the bootloader, as stated on https://forum.quicklogic.com/viewtopic.php?t=29. This is for the "original" Quickfeather Quicklogic, but this guide is compatible. However, there is one error on the page. The git repository which contains the factory bootloader uses the name "qf_loadflash.bin" not "ql_loadflash.bin" as stated in the guide. Following the rest of the guide solved the problem for all three boards.

For those who don't have a JLink debugger and are thinking of using OpenOCD. I have tried this with a STLink V2 and got assistance in the IRC channel from someone more knowledgable. Currently, the OpenOCD drivers only support debugging and not flashing. There is no functional flash driver for the EOS S3 chip as of now (it doens't recognize the chip), which would enable OpenOCD to flash the EOS S3. This makes me believe that a JLink device is required to recover the boards.
